### PR TITLE
[4.0] Change color to have sufficient contrast as per a11y standards

### PR DESCRIPTION
**Pull request by hardik-codes**
Pull Request for Issue #24261 

### Testing Instructions

Test the administrator login page using the following tool http://aka.ms/AccessibilityInsights.

### Expected result
No contrast error

### Actual result
contrast error

### Documentation Changes Required
None

### CODE REVIEW

File: administrator/templates/atum/scss/blocks/_login.scss [hunk 1]
```diff
@@ -58,7 +58,7 @@
 
       span {
         font-size: .9rem;
-        color: #c6d2de;
+        color: #CBD8E1;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_login.scss]: Please add this to the `variables.scss` and make the hex all lowercase

**Review by C-Lodder — CHANGES_REQUESTED** [review]

File: administrator/templates/atum/scss/blocks/_login.scss (hunk 1 above)
Comment (hardik-codes) [reply administrator/templates/atum/scss/blocks/_login.scss]: @C-Lodder in which line should I add the variable for this color?

**Comment by hardik-codes** [comment]
@C-Lodder Now have a look

**Review by C-Lodder — APPROVED** [review]

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**